source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -170,7 +170,8 @@ Decommissioning Ignores Expired Objects and Trailing ``DeleteMarker``
 Starting with :minio-release:`RELEASE.2023-05-27T05-56-19Z`, decommissioning ignores objects where the only remaining version is a ``DeleteMarker``.
 This avoids creating empty metadata on the remaining server pool(s) for objects that are effectively fully deleted.
 
-Starting with :minio-release:`minio-lifecycle-management-scanner`, decommissioning also ignores object versions which have expired based on the configured :ref:`lifecycle rules <minio-lifecycle-management-expiration>` for the parent bucket.
+Starting with :minio-release:`RELEASE.2023-06-23T20-26-00Z`, decommissioning also ignores object versions which have expired based on the configured :ref:`lifecycle rules <minio-lifecycle-management-expiration>` for the parent bucket.
+Starting with :minio-release:`RELEASE.2023-06-29T05-12-28Z`, decommissioning logs ignored delete markers and expired objects.
```

Comment:
```suggestion
Starting with :minio-release:`RELEASE.2023-06-29T05-12-28Z`, decommissioning-related logs ignore delete markers and expired objects.
```

Just to avoid the concept of decommissioning logs being distinct from the system logs (which I might be wrong about but I think it's all one log)

Reply:
I'm gonna merge this with your suggestion, can update to make less vague if/when new details surface.

Reply:
reworked because not logging